src/core/actions/creators.ts

```diff
@@ -316,12 +316,7 @@ namespace ActionCreators {
   export function updateQuery(query: string): Actions.ResetPageAndUpdateQuery {
     return [
       ActionCreators.resetPage(),
-      createAction(Actions.UPDATE_QUERY, query && query.trim(), {
-        payload: [
-          validators.isValidQuery,
-          validators.isDifferentQuery
```

Comment:
Find out if anything else uses `isDifferentQuery` and delete it + tests if it's not used anymore